modules/exploits/linux/games/ut2004_secure.rb

```diff
@@ -89,24 +96,20 @@ def ut_version
   def check
     vers = ut_version
 
-    if (not vers)
-      vprint_status("Could not detect Unreal Tournament Server")
-      return Exploit::CheckCode::Unknown
+    if !vers
+      return CheckCode::Safe('Could not detect Unreal Tournament Server')
     end
 
     print_status("Detected Unreal Tournament Server Version: #{vers}")
+
     if (vers =~ /^(3120|3186|3204)$/)
-      vprint_status("This system appears to be exploitable")
-      return Exploit::CheckCode::Appears
+      return Exploit::CheckCode::Appears('This system appears to be exploitable')
     end
 
-
     if (vers =~ /^(2...)$/)
-      vprint_status("This system appears to be running UT2003")
-      return Exploit::CheckCode::Detected
+      return CheckCode::Detected('This system appears to be running UT2003')
```

Comment:
Let's keep these `CheckCode` and `Exploit::CheckCode` calls consistent. Let's use one or the other

Reply:
changed